Mathlib/Algebra/BigOperators/Field.lean

```diff
@@ -20,3 +21,33 @@ lemma Multiset.sum_map_div (s : Multiset ι) (f : ι → K) (a : K) :
 
 lemma Finset.sum_div (s : Finset ι) (f : ι → K) (a : K) :
     (∑ i ∈ s, f i) / a = ∑ i ∈ s, f i / a := by simp only [div_eq_mul_inv, sum_mul]
+
+-- TODO: Move these once `Finset.dens` doesn't depend on `Field` anymore
```

Comment:
Where do you want to move them to? Mightn't it be easier to just put them in a new file now?

Reply:
to `Algebra.BigOperators.Group.Finset.Basic`, next to the corresponding `card` lemmas

Reply:
Please mention that in your comment.